install-deps.sh

```diff
@@ -168,7 +168,7 @@ function install_pkg_on_ubuntu {
 
 function install_boost_on_ubuntu {
     in_jenkins && echo "CI_DEBUG: Running install_boost_on_ubuntu() in install-deps.sh"
-    local ver=1.75
+    local ver=1.79
```

Comment:
this does not work if we don't have precompiled boost packages uploaded to chacra.

Reply:
See also https://github.com/ceph/ceph/commit/89121189ac9604913a2a17dc1a224e64cfdde3a1

I will upload the tarball.

Reply:
Oh, sorry, I pushed the tarball to download.ceph.com.  I'll look into the other bit.